cms/templates/js/certificate-editor.underscore

```diff
@@ -25,6 +25,13 @@
                 <span class="tip tip-stacked"><%= gettext("Optional long description") %></span>
             </div>
         </fieldset>
+         <header>
+           <label class="title-2"><%= gettext("Course Signatories") %></label>
+         </header>
+        <div class="signatory-list"> </div>
+        <span>
+           <a class="button action-add-signatory" href="javascript:void(0);"><%= gettext("Add Signatory") %></a>
```

Comment:
We should indicate the limit here -- perhaps it could say "Add Signatory (max 4)"